protocol/export_test.go

```diff
@@ -0,0 +1,9 @@
+package protocol
+
+func SetChainHeight(c *Chain, height uint64) {
+	c.setHeight(height)
+}
+
+func GetChainHeight(c *Chain) uint64 {
+	return c.state.height
+}
```

Comment:
Can we use c.Height instead? Or do we need to explicitly not lock the protecting mutex?